### PR TITLE
Added visibility to welcome message

### PR DESCRIPTION
**Pull request by jneelamekam**

### CODE REVIEW

File: src/dispatch/incident/messaging.py [hunk 1]
```diff
@@ -66,6 +66,7 @@ def get_suggested_documents(db_session, incident: Incident) -> list:
 def send_welcome_ephemeral_message_to_participant(
     participant_email: str, incident: Incident, db_session: SessionLocal
 ):
+
```
Comment (wssheldon) [inline src/dispatch/incident/messaging.py:69]:
```suggestion
```

File: src/dispatch/messaging/strings.py [hunk 2]
```diff
@@ -34,6 +35,10 @@ class MessageType(DispatchEnum):
     IncidentStatus.closed: "This no longer requires additional involvement, long term incident action items have been assigned to their respective owners.",
 }
 
+INCIDENT_VISIBILITY_DESCRIPTIONS = {
+    Visibility.open: " We ask that you use your best judgment while sharing details about this incident outside of the dedicated channels of communication.  Please reach out to the Incident Commander if you have any questions.",
+    Visibility.restricted: "This incident is restricted to immediate participants of this incident. We ask that you exercise extra caution and discretion while talking about this incident outside of the dedicated channels of communication , and only invite new participants that are strictly necessary. Please reach out to the Incident Commander if you have any questions.",
```
Comment (wssheldon) [inline src/dispatch/messaging/strings.py:40]:
```suggestion
    Visibility.restricted: "This incident is restricted to immediate participants of this incident. We ask that you exercise extra caution and discretion while talking about this incident outside of the dedicated channels of communication. Only invite new participants that are strictly necessary. Please reach out to the Incident Commander if you have any questions.",
```

File: src/dispatch/messaging/strings.py [hunk 3]
```diff
@@ -34,6 +35,10 @@ class MessageType(DispatchEnum):
     IncidentStatus.closed: "This no longer requires additional involvement, long term incident action items have been assigned to their respective owners.",
 }
 
+INCIDENT_VISIBILITY_DESCRIPTIONS = {
+    Visibility.open: " We ask that you use your best judgment while sharing details about this incident outside of the dedicated channels of communication.  Please reach out to the Incident Commander if you have any questions.",
```
Comment (wssheldon) [inline src/dispatch/messaging/strings.py:39]:
```suggestion
    Visibility.open: "We ask that you use your best judgment while sharing details about this incident outside of the dedicated channels of communication.  Please reach out to the Incident Commander if you have any questions.",
```

File: src/dispatch/messaging/strings.py [hunk 4]
```diff
@@ -747,6 +758,8 @@ def render_message_template(message_template: List[dict], **kwargs):
 
                 if button.get("button_url"):
                     button["button_url"] = Template(button["button_url"]).render(**kwargs)
+        if d.get("visibility_mapping"):
```
Comment (wssheldon) [inline src/dispatch/messaging/strings.py:761]:
```suggestion

        if d.get("visibility_mapping"):
```

**Review by wssheldon — COMMENTED** [review]
Looks good, couple of typo/style suggestions.

File: src/dispatch/messaging/strings.py [hunk 5]
```diff
@@ -34,6 +35,10 @@ class MessageType(DispatchEnum):
     IncidentStatus.closed: "This no longer requires additional involvement, long term incident action items have been assigned to their respective owners.",
 }
 
+INCIDENT_VISIBILITY_DESCRIPTIONS = {
+    Visibility.open: "We ask that you use your best judgment while sharing details about this incident outside of the dedicated channels of communication.  Please reach out to the Incident Commander if you have any questions.",
```
Comment (mvilanova) [inline src/dispatch/messaging/strings.py:39]:
```suggestion
    Visibility.open: "We ask that you use your best judgment while sharing details about this incident outside of the dedicated channels of communication. Please reach out to the Incident Commander if you have any questions.",
```

File: src/dispatch/incident/messaging.py [hunk 6]
```diff
@@ -307,6 +309,7 @@ def send_incident_created_notifications(incident: Incident, db_session: SessionL
         "severity_description": incident.incident_severity.description,
         "priority": incident.incident_priority.name,
         "priority_description": incident.incident_priority.description,
+        "visibility": incident.visibility,
```
Comment (mvilanova) [inline src/dispatch/incident/messaging.py:312]: @jneelamekam should we bump it further up in the message to ensure people read the description? Perhaps between description and status?

Comment (jneelamekam) [reply src/dispatch/incident/messaging.py]: Updated the position of where we display changes in the recent commit

**Pull request merged**